coins/testnet/zel.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "zelcash",
+    "symbol": "zel",
+    "algorithm": "equihash",
+    "parameters": {
+        "N": 125,
+        "K": 4,
+        "personalization": "ZelProof"
+    },
+    "requireShielding": true,
+    "txfee": 0.0001,
+    "peerMagic": "24e92764",
```

I believe this change is needed:
```suggestion
    "peerMagic": "fa1af9bf",
```